### PR TITLE
French translation update [Issue #333]

### PR DESCRIPTION
**Pull request by BRUCELLA2**
## Status
**READY**

## Migrations
NO

## Description
French translation update

### CODE REVIEW

**Comment by mcpierce** [comment]
@BRUCELLA2 Can you rebase this PR so that it's just the translation, please?

**Comment by BRUCELLA2** [comment]
> @BRUCELLA2 Can you rebase this PR so that it's just the translation, please?

Yes, i can :+1:

File: comixed-frontend/src/assets/i18n/fr/comics.json [hunk 1]
```diff
@@ -181,36 +181,35 @@
...
+        "page-type": "Type de la page",
+        "hash": "Hash de la page",
+        "dimensions": "Taille",
+        "blocked-status": "Bloquée",
+        "deleted-status": "supprimée"
       },
       "column": {
-        "blocked-status": "{blocked, select, true{Yes} other{No}}",
-        "deleted-status": "{deleted, select, true{Yes} other{No}}"
+        "blocked-status": "{bloquée, sélectionner, true{Oui} other{Non}}",
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/fr/comics.json]: In this and the following line, the keyword "select" was translated, and needs to be reverted. Also the variables "blocked" and "deleted" were translated and need to be reverted.

File: comixed-frontend/src/assets/i18n/fr/comics.json [hunk 2]
```diff
@@ -181,36 +181,35 @@
...
+      "letters": "Page de courriers",
+      "preview": "Aperçu",
+      "other": "Autre",
+      "filtered": "Filtré"
     },
     "tooltip": {
-      "blocked": "Click to {blocked, select, true{unblock} other{block}} this page type...",
-      "deleted": "Click to {deleted, select, true{undelete} other{delete}} this page..."
-    }
+      "blocked": "Cliquer pour {bloquer, sélectionner, true{débloquer} other{bloquer}} le type de cette page...",
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/fr/comics.json]: Here and in the following line the key word "select" was translated and needs to be reverted. Also the variables "blocked" and "deleted" were translated and need to be reverted.

File: comixed-frontend/src/assets/i18n/fr/library.json [hunk 3]
```diff
@@ -1,36 +1,36 @@
...
+        "COVERDATE": "La date de couverture pour cette publication"
       },
-      "delete-files-warning-header": "Delete Physical Files",
-      "delete-files-warning-message": "Deleting physical files cannot be undone. Are you SURE you want to do this?",
-      "confirmation-header": "Organize Library",
-      "confirmation-message": "Are you sure you want to {deletePhysicalFiles, select, true{delete comic files marked for deletion and} other{}} organizing the library?"
+      "delete-files-warning-header": "Suppression des fichiers physiques",
+      "delete-files-warning-message": "La suppression des fichiers physiques ne peut pas être annulée. Êtes-vous sûr de vouloir le faire ?",
+      "confirmation-header": "Organiser la bibliothèque",
+      "confirmation-message": "Êtes-vous sûr de vouloir {effacer les fichiers physiques, sélectionner, true {effacer les fichiers dex bande dessinée marqués pour suppression et} autres { et}} organiser la bibliothèque ?"
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/fr/library.json]: The variable "deletePhysicalFiles" was translated and needs to be reverted.

The keyword "other" was translated here and needs to be reverted. Also there shouldn't be a space between the keyword and the opening curly brace.

**Review by mcpierce — CHANGES_REQUESTED** [review]
Found a few issues with keywords and variable names being translated. I also saw an entry that wasn't translated in comics.json:

```"max-records": "Max Records To Fetch"```

Could you translate that line as well? And please be sure to test the translation locally.

File: comixed-frontend/src/assets/i18n/fr/comics.json (hunk 1 above)
Comment (BRUCELLA2) [reply comixed-frontend/src/assets/i18n/fr/comics.json]: fixed

File: comixed-frontend/src/assets/i18n/fr/comics.json (hunk 2 above)
Comment (BRUCELLA2) [reply comixed-frontend/src/assets/i18n/fr/comics.json]: fixed

File: comixed-frontend/src/assets/i18n/fr/library.json (hunk 3 above)
Comment (BRUCELLA2) [reply comixed-frontend/src/assets/i18n/fr/library.json]: fixed

**Comment by mcpierce** [comment]
@BRUCELLA2 Is this rebased on the latest develop?

**Review by mcpierce — APPROVED** [review]

**Pull request merged**